runtime/calamari/src/lib.rs

```diff
@@ -82,7 +84,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("calamari"),
 	impl_name: create_runtime_str!("calamari"),
 	authoring_version: 1,
-	spec_version: 1,
+	spec_version: 2,
```

Comment:
Since we are not changing the genesis this needs to be 2 and the testnet has to be upgraded after launch to introduce the new runtime.
If we launch a testnet with 1 the new pallets will show up in the front-end but their extrinsics will fail.